### PR TITLE
Create do-you-speak-prestashop-may-2023.md

### PR DESCRIPTION
**Pull request by l-delin**
Here's the May '23 Crowdin Monthly! 🗞

ping @coffrep @kpodemski for review

### CODE REVIEW

**Review by coffrep — APPROVED** [review]

**Comment by l-delin** [comment]
Thanks @coffrep! 
Quick edit, I just added a link to the QA checker doc to the article.

**Review by coffrep — APPROVED** [review]

**Comment by coffrep** [comment]
Good for me @l-delin !

File: content/news/translation/2023/do-you-speak-prestashop-may-2023.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+aliases: ["/news/do-you-speak-prestashop-may-2023"]
+title: "Do You Speak PrestaShop? – May 2023 Edition"
+subtitle: "Where you'll learn that PrestaShop 8.1 is ready to be translated"
+date: 2023-06-9 10:00:00
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-may-2023.md]:
```suggestion
date: 2023-06-09
```

File: content/news/translation/2023/do-you-speak-prestashop-may-2023.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+aliases: ["/news/do-you-speak-prestashop-may-2023"]
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-may-2023.md]:
```suggestion
```

no need

**Comment by l-delin** [comment]
Thanks!

**Review by kpodemski — APPROVED** [review]

**Pull request merged**